templates/cisco_asa_show_interface.textfsm

```diff
@@ -25,12 +25,14 @@ Value FIVEMIN_DROP_RATE (\d+)
 
 Start
   ^Interface.* -> Continue.Record
-  ^.*Interface ${INTERFACE} "${INTERFACE_ZONE}", is ${LINK_STATUS}.*protocol is ${PROTOCOL_STATUS}
+  ^.*Interface ${INTERFACE} "${INTERFACE_ZONE}", is ${LINK_STATUS},.*protocol\s+is\s+${PROTOCOL_STATUS}
+  ^.*Interface ${INTERFACE}.*is\s+${LINK_STATUS},.*protocol\s+is\s+${PROTOCOL_STATUS}
```

Comment:
I would probably get rid of as many of the `.*` as possible in this line such as this:

```suggestion
  ^.*Interface ${INTERFACE}.*is\s+${LINK_STATUS},\s+line\s+protocol\s+is\s+${PROTOCOL_STATUS}
```

Reply:
Makes sense. I wanted to get something out there. Hopefully I can cut it back. We only have one other example thus far. Hopefully it doesn't break anything.